src/turnkeyml/version.py

```diff
@@ -1 +1 @@
-__version__ = "4.0.0"
+__version__ = "4.0.1"
```

Comment:
MINOR: Incremented when adding new features or functionality in a backward-compatible manner.
```suggestion
__version__ = "4.1.0"
```

Reply:
That's not how we define MINOR in https://github.com/onnx/turnkeyml/blob/main/docs/versioning.md